### PR TITLE
Translate multiple release posts (zh_tw)

### PR DESCRIPTION
**Pull request by bugtender**
zh_tw Translation for multiple release posts

  - zh_tw/news/_posts/2017-09-14-json-heap-exposure-cve-2017-14064.md
  - zh_tw/news/_posts/2017-09-14-openssl-asn1-buffer-underrun-cve-2017-14033.md
  - zh_tw/news/_posts/2017-09-14-ruby-2-2-8-released.md
  - zh_tw/news/_posts/2017-09-14-ruby-2-3-5-released.md
  - zh_tw/news/_posts/2017-09-14-ruby-2-4-2-released.md
  - zh_tw/news/_posts/2017-09-14-sprintf-buffer-underrun-cve-2017-0898.md
  - zh_tw/news/_posts/2017-09-14-webrick-basic-auth-escape-sequence-injection-cve-2017-10784.md

Thank you.

### CODE REVIEW

File: zh_tw/news/_posts/2017-09-14-json-heap-exposure-cve-2017-14064.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+lang: zh_tw
+---
+
+Ruby 內建的 JSON 含有堆積暴露的安全性風險，此風險的 CVE 識別號已被指派為 [CVE-2017-14064](http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2017-14064)。
+
+## 詳情
+
+`JSON` 模組的 `generate` 方法可選擇性地接受 `JSON::Ext::Generator::State` 類別的實體，如果傳遞了惡意的實體，結果可能會包含堆積的內容。
+
+所有正在使用受影響版本的使用者應立即升級或是使用採用應急辦法。
```
Comment (JuanitoFatas) [inline zh_tw/news/_posts/2017-09-14-json-heap-exposure-cve-2017-14064.md]: 使用採用?

File: zh_tw/news/_posts/2017-09-14-openssl-asn1-buffer-underrun-cve-2017-14033.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+lang: zh_tw
+---
+
+Ruby 內建的 OpenSSL 有緩衝區掏空的安全性風險，此風險的 CVE 識別號已被指派為 [CVE-2017-14033](http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2017-14033)。
+
+## 詳細
+
+如果對 `OpenSSL::ASN1` 的 `decode` 方法傳遞惡意字串，緩衝區掏空可能會造成 Ruby 直譯器崩潰。
+
+所有正在使用受影響版本的使用者應立即升級或是使用採用應急辦法。
```
Comment (JuanitoFatas) [inline zh_tw/news/_posts/2017-09-14-openssl-asn1-buffer-underrun-cve-2017-14033.md]: 使用採用?

File: zh_tw/news/_posts/2017-09-14-ruby-2-4-2-released.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+layout: news_post
+title: "Ruby 2.4.2 發布"
+author: "nagachika"
+translator: "Vincent Lin"
+date: 2017-09-14 00:00:00 +0000
+lang: zh_tw
+---
+
+我們很榮幸的
+We are pleased to announce the release of Ruby 2.4.2.
```
Comment (JuanitoFatas) [inline zh_tw/news/_posts/2017-09-14-ruby-2-4-2-released.md]: missing translation 😅

File: zh_tw/news/_posts/2017-09-14-webrick-basic-auth-escape-sequence-injection-cve-2017-10784.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+Ruby 內建的 WEBrick 基本認證有轉義序列注入的安全性風險。
+此風險的 CVE 識別號已被指派為 [CVE-2017-10784](http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2017-10784) 。
+
+## 詳細
+
+當使用 WEBrick 的基本認證時，客戶端可以傳入任意字串作為使用者名稱。
+WEBrick 完整地將使用者名稱紀錄至紀錄檔（log）中，而攻擊者可通過紀錄檔注入惡意的轉義序列進而操縱受害者在終端機（terminal）執行危險的控制字符。
+
+此安全性風險與[另一個已修復的風險](/en/news/2010/01/10/webrick-escape-sequence-injection/)相似，但在基本認證中並沒有修復。
```
Comment (JuanitoFatas) [inline zh_tw/news/_posts/2017-09-14-webrick-basic-auth-escape-sequence-injection-cve-2017-10784.md]: `/en/news/` do we have a chinese translation?

Comment (bugtender) [reply zh_tw/news/_posts/2017-09-14-webrick-basic-auth-escape-sequence-injection-cve-2017-10784.md]: this post only have en/ja/ko version 😭

Comment (JuanitoFatas) [reply zh_tw/news/_posts/2017-09-14-webrick-basic-auth-escape-sequence-injection-cve-2017-10784.md]: We can translate in future PRs :)

**Comment by JuanitoFatas** [comment]
Thanks for the contributions!!! 👍 👍 👍 🎉

**Pull request merged**